tests/schema.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+    "root": {
```

Comment:
Should these top level keys be unique files?

It's not clear looking at the file in isolation what is being checked.

Alternatively I think these can all be just joined together using referencing: https://ajv.js.org/guide/combining-schemas.html